src/logic/pluginLibraryLogic.js

```diff
@@ -29,10 +26,18 @@ export const pluginLibraryLogic = kea({
         pluginLoading: [
             false,
             {
-                setPluginLoading: (_, { pluginLoading }) => pluginLoading,
+                [actions.openPlugin]: () => true,
+                [actions.setActivePlugin]: () => false,
             },
         ],
-    },
+        modalOpen: [
+            false,
+            {
+                [actions.openPlugin]: () => true,
+                [actions.closeModal]: () => false,
+            },
+        ],
```

Comment:
instead of just true/false, store the name of the opened plugin in a reducer. it'll help with routing later on